docs/content/entity-templates.md

```diff
@@ -53,6 +53,54 @@ public static class CreatureTemplate
 
 You can use this entity template to spawn a Creature entity. This can be done using the [MonoBehaviour](gameobject/world-commands.md) or [ECS](ecs/world-commands.md) workflow.
 
+### Feature Module Components
+
+To take advantage of some feature modules, you need to add the SpatialOS components that these modules will make use of. This is done by calling the appropriate helper method for each desired feature module.
+
+#### Transform Synchronization
+
+The helper for this feature module requires the **attribute** of the worker you want to give authority over Transforms to. There is also the *option* to set a starting **location**, **velocity** and **rotation**. If not provided, location and velocity will be set to `default(Vector3)`, and rotation will be set to `Quaternion.identity`.
+
+```c#
+public static class CubeTemplate
+{
+    public static Entity CreateCubeEntityTemplate(Coordinates coords)
+    {
+        var entityBuilder = EntityBuilder.Begin()
+            .AddPosition(coords.X, coords.Y, coords.Z, WorkerUtils.UnityGameLogic)
+            ...
+            ...
+            .AddTransformSynchronizationComponents(WorkerUtils.UnityGameLogic,
+				location: coords.NarrowToUnityVector());
+
+        return entityBuilder.Build();
+    }
+}
+```
+
+#### Player Lifecycle
+
+This helper for this feature module requires a **clientAttribute** and a **serverAttribute**.
+
+```c#
+public static class PlayerTemplate
+{
+    public static Entity CreatePlayerEntityTemplate(List<string> clientAttributeSet, Improbable.Vector3f position)
+    {
+        // Obtain unique client attribute
+        var clientAttribute = clientAttributeSet.First(attribute => attribute != WorkerUtils.UnityClient);
+
+        var entityBuilder = EntityBuilder.Begin()
+            .AddPosition(position.X, position.Y, position.Z, WorkerUtils.UnityGameLogic)
+            ...
+            ...
+            .AddPlayerLifecycleComponents(clientAttribute, WorkerUtils.UnityGameLogic);
+
+        return entityBuilder.Build();
+    }
+}
+```
+
```

Comment:
The whole doc needs a once-over as it's got a lot of unclear passive voice and prophecies about the future (will) rather than statements of fact - I know you didn't write that, though.
There's quite a few bits and bobs in this diff, so I am going to go through the whole thing here - I am adding my edits to the text, below in here and added questions in "<Editor questions:....>".

Notes to explain the edits:
We use newspaper titles (sentences without full stops) for our title case.
"Feature Module(s)" and "Core Module" are proper nouns (blame Callum!).
We avoid "will" wherever possible because it reads like a prophecy for the future (and implies instability).
We avoid passive voice as it doesn't make it clear who does the action - you can tell passive voice by adding "by zombies" to a sentence. If the sentence makes sense with "by zombies" attached to it, it's in passive voice (IE "This is done by zombies calling the appropriate helper method".
We only use the bold (**) for titles, to call out "Note:" as well as for anything on a UI.

-----

### Feature Module components
 To take advantage of some Feature Modules, you need to add the SpatialOS components that these modules make use of. You do this by calling the appropriate helper method for each feature module. This document lists the helper methods and how to use them.

_<Editor questions: why "some" feature modules?>_
_<Editor questions: add the components to what or where?>_
_<Editor questions: calling the appropriate from where?>_
_<Editor questions: "This document lists the helper methods and how to use them." Is this right?>_

 #### Transform synchronization module
When you want to give a [worker](./worker.md) authority [authority](LINK!TBD) over the transform synchronisation of a SpatialOS component, you use the _<Editor questions: name?>_ helper method. This Feature Module requires the attribute of the worker you are giving authority to.
_<Editor questions: where do we find out about attributes?>_
You have the option to set a starting `location`, `velocity` and `rotation`. If you do not provide these, location and velocity are `default(Vector3)`, and rotation is `Quaternion.identity`.

Example showing _<Editor questions: what are we showing here?>_

```c sharp
public static class CubeTemplate
{
    public static Entity CreateCubeEntityTemplate(Coordinates coords)
    {
        var entityBuilder = EntityBuilder.Begin()
            .AddPosition(coords.X, coords.Y, coords.Z, WorkerUtils.UnityGameLogic)
            ...
            ...
            .AddTransformSynchronizationComponents(WorkerUtils.UnityGameLogic,
				location: coords.NarrowToUnityVector());
         return entityBuilder.Build();
    }
}
```
 #### Player lifecycle
 The helper method _<Editor questions: what's it called?>_ for this Feature Module _<Editor questions: does what?>_ . It requires a `clientAttribute` and a `serverAttribute`.
 
Example showing _<Editor questions: what are we showing here?>_
```c sharp
public static class PlayerTemplate
{
    public static Entity CreatePlayerEntityTemplate(List<string> clientAttributeSet, Improbable.Vector3f position)
    {
        // Obtain unique client attribute
        var clientAttribute = clientAttributeSet.First(attribute => attribute != WorkerUtils.UnityClient);
         var entityBuilder = EntityBuilder.Begin()
            .AddPosition(position.X, position.Y, position.Z, WorkerUtils.UnityGameLogic)
            ...
            ...
            .AddPlayerLifecycleComponents(clientAttribute, WorkerUtils.UnityGameLogic);
         return entityBuilder.Build();
    }
}
```